WalletWasabi/Exceptions/InsufficientBalanceException.cs

```diff
@@ -5,7 +5,7 @@ namespace WalletWasabi.Exceptions
 {
 	public class InsufficientBalanceException : Exception
 	{
-		public InsufficientBalanceException(Money minimum, Money actual) : base($"Needed: {minimum.ToString(false, true)} BTC, got only: {actual.ToString(false, true)} BTC.")
+		public InsufficientBalanceException(Money minimum, Money actual) : base($"Needed: {minimum.ToString(false, true)} BTC for this transaction but only: {actual.ToString(false, true)} BTC was selected to send")
```

Comment:
It's better. A couple of things:

1. Don't you need a comma before the but?
2. "was selected" vs "is selected"?
3. Could you put a dot to the end of the sentence?
4. Could you remove the two : character?

Reply:
1.  Comma added. Simple grammar mistake
2.  If we use "Needed" I think "was" matches the past tense
3.  Period added.
4.  Removed colons (Ouch!)

Because this is an error message maybe it shouldn't be in the past tense. I am uploading a present tense version as:
"Need X BTC for this transaction, but only Y BTC is selected to send."

![bettererrorfeedback](https://user-images.githubusercontent.com/44556677/47969634-dbc8e780-e037-11e8-8a47-9b6299331942.png)